backend/src/main/resources/static/semantic-hub-openapi.yaml

```diff
@@ -128,6 +128,49 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /models/bulk:
+    post:
+      tags:
+        - SemanticHub
+      operationId: getModelListByUrns
+      parameters:
```

Comment:
To stay consistent with the GET /models endpoint, one could add an optional filter parameter for the status. For me it is ok, to do this in a follow-up PR and not in this initial work for the bulk endpoint.

Reply:
Good point! As suggested, this will be done in a separate PR.